build.gradle

```diff
@@ -22,9 +22,7 @@ repositories {
 
 dependencies {
     implementation('net.portswigger.burp.extender:burp-extender-api:2.3')
-    // retrofit
-    implementation("com.squareup.retrofit2:retrofit:2.9.0")
-    implementation("com.squareup.retrofit2:converter-gson:2.9.0")
+    implementation('com.fasterxml.jackson.core:jackson-databind:2.12.3')
```

Comment:
@mgiamberardino do you think this version might have any conflicts with existing libs in Burp? Do they have any JSON serializer already being used?

Reply:
@buchi-busireddy I guess that could happen either with gson or jackson. Maybe we should let the support team us now.